java/com/intel/daal/utils/LibUtils.java

```diff
@@ -140,7 +140,7 @@ public static String createLibraryFileName(String name) throws IOException
 
         if (OSname.startsWith("linux")) {
             if (name.contains("tbb")) {
-                fullName = "lib" + name + ".so.2";
+                fullName = "lib" + name + ".so.12";
```

Comment:
@xwu99, have you encountered similar problem with libtbbmalloc.so.2? If yes, there is probably more correct fix:
```suggestion
                if (name.contains("malloc")) {
                    fullName = "lib" + name + ".so.2";
                }
                else {
                    fullName = "lib" + name + ".so.12";
                }
```